### PR TITLE
[FEAT] User review prompt

### PR DESCRIPTION
**Pull request by gantunesr**

### CODE REVIEW

File: app/components/Nav/Main/index.js [hunk 1]
```diff
@@ -808,4 +811,17 @@ const mapDispatchToProps = (dispatch) => ({
...
+const ConnectedMain = connect(mapStateToProps, mapDispatchToProps)(Main);
+
+const MainFlow = () => (
+	<Stack.Navigator
+		initialRouteName={'Main'}
+		mode={'modal'}
+		screenOptions={{ headerShown: false, cardStyle: { backgroundColor: 'transparent' } }}
+	>
+		<Stack.Screen name={'Main'} component={ConnectedMain} />
+		<Stack.Screen name={'ReviewModal'} component={ReviewModal} options={{ animationEnabled: false }} />
```
Comment (sethkfman) [inline app/components/Nav/Main/index.js:823]: Any specific reason why animation is disabled?

File: app/components/UI/ReusableModal/styles.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { StyleSheet } from 'react-native';
+import { colors } from '../../../styles/common';
+
+export default StyleSheet.create({
```
Comment (sethkfman) [inline app/components/UI/ReusableModal/styles.ts:4]: I see that the style has been separated from the `index.tsx` file. Is this a new pattern that we are going to be using?

Comment (Cal-L) [reply app/components/UI/ReusableModal/styles.ts:4]: One thing I've noticed that bloats the component files are styles. While this isn't necessary, I'd like to begin isolating styles into their own files to help alleviate the extra noise + clean up the component files. This pattern can work well outside of styles as well such as separate business logic, hooks, etc. WDYT?

Comment (rickycodes) [reply app/components/UI/ReusableModal/styles.ts:4]: there's probably an opportunity to merge/share a lot of styles this way as well

**Review by cortisiko — APPROVED** [review]
🌮 🌮 🌮 🌮

**Review by sethkfman — APPROVED** [review]
LGTM

**Pull request merged**